code/game/turfs/turf.dm

```diff
@@ -528,13 +528,15 @@
 /turf/AllowDrop()
 	return TRUE
 
-/turf/proc/add_vomit_floor(mob/living/carbon/M, toxvomit = 0)
+/turf/proc/add_vomit_floor(mob/living/carbon/M, toxvomit = FALSE, purpvomit = FALSE)
```

Comment:
let's change this a bit, there's no point in having 2 args that are basically both options that can't ever overlap.

`/turf/proc/add_vomit_floor(mob/living/carbon/M, toxvomit = NONE) //NONE already exists as a define`

Reply:
isn't NONE specifically for bitfields

Reply:
I think it's used in bitfields as well, dunno if the define is exclusive to bitfields though.